disco_aws_automation/disco_elastigroup.py

```diff
@@ -272,13 +304,50 @@ def delete_groups(self, hostclass=None, group_name=None, force=False):
         """Delete all elastigroups based on hostclass"""
         # We need argument `force` to match method in autoscale
         # pylint: disable=unused-argument
-        groups = self._get_existing_groups(hostclass=hostclass, group_name=group_name)
+        groups = self.get_existing_groups(hostclass=hostclass, group_name=group_name)
         for group in groups:
             logger.info("Deleting group %s", group['name'])
             self._delete_group(group_id=group['id'])
 
+    def scaledown_groups(self, hostclass=None, group_name=None, wait=False, noerror=False):
+        """
+        Scales down number of instances in a hostclass's elastigroup, or the given elastigroup, to zero.
+        If wait is true, this function will block until all instances are terminated, or it will raise
+        a WaiterError if this process times out, unless noerror is True.
+
+        Returns true if the elastigroups were successfully scaled down, False otherwise.
+        """
+        groups = self.get_existing_groups(hostclass=hostclass, group_name=group_name)
+        for group in groups:
+            group_update = {
+                "group": {
+                    "capacity": {
+                        "target": 0,
+                        "minimum": 0,
+                        "maximum": 0
+                    }
+                }
+            }
+            logger.info("Scaling down group %s", group['name'])
+            self.session.put(SPOTINST_API + group['id'], data=json.dumps(group_update))
```

Comment:
Maybe a helper method like `self._send_request(group['id'], group_update` would be useful

```python
def _send_request(url, data):
    self.session.put(SPOTINST_API + url, data=json.dumps(data))
```